### PR TITLE
[feature] add new feature to Find Dialog: "Count Matches"

### PR DESCRIPTION
**Pull request by ivandrofly**
# Status
- [x] Regex texted
     - [x] Valid regex.
     - [x] Invalid regex.
- [x] Large subtitle file (10k lines)

### CODE REVIEW

File: src/Forms/FindDialog.cs [hunk 1]
```diff
@@ -178,9 +180,16 @@ internal void SetIcon(Bitmap bitmap)
             if (bitmap != null)
             {
                 IntPtr Hicon = bitmap.GetHicon();
-                this.Icon = Icon.FromHandle(Hicon);
```
Comment (ivandrofly) [inline src/Forms/FindDialog.cs]: Couldn't leave this ;)

**Comment by niksedk** [comment]
Rebase?

Have you tested with really large subtitle + regex?

**Comment by ivandrofly** [comment]
> Rebase?
> 
> Have you tested with really large subtitle + regex?

:+1: Check status

**Comment by ivandrofly** [comment]
@niksedk  can be merged

**Comment by niksedk** [comment]
Does this work when just hitting ctrl+f and typing something? (I could not see anything)

**Comment by ivandrofly** [comment]
> Does this work when just hitting ctrl+f and typing something? (I could not see anything)

:+1:

**Comment by niksedk** [comment]
I could not see anything...

**Comment by ivandrofly** [comment]
> I could not see anything...

What do you mean? o.O

**Comment by ivandrofly** [comment]
@niksedk can you try again?

**Comment by niksedk** [comment]
What steps do I do to see anything? Still cannot see any info / text!?

**Comment by ivandrofly** [comment]
> What steps do I do to see anything? Still cannot see any info / text!?

You will have enter text and click count it will display how many repetition exist in current subtitle.

**Comment by niksedk** [comment]
Ah, works nicely :)
Could you add the two localization texts also?

**Comment by niksedk** [comment]
and the button text...

**Comment by ivandrofly** [comment]
> and the button text...

Done

**Comment by xylographe** [comment]
Multi-lang `FindDialog` texts:
```
    WholeWord = "Whole word",
    Count = "Count",
    XNumberOfMatches = "{0} matches"
```

**Comment by ivandrofly** [comment]
:+1:  @xylographe

**Comment by xylographe** [comment]
**UpdateLanguageFiles** is still generating `src\Logic\LanguageDeserializer.cs` instead of `libse\LanguageDeserializer.cs`.

I fixed `libse\LanguageDeserializer.cs` manually, but this needs to be fixed in `build_helpers.bat` and `src\UpdateLanguageFiles\LanguageDeserializerGenerator.cs`.

**Comment by xylographe** [comment]
I think, the order of the buttons could be

- Find
- Count
- Cancel

Or, better, only **Find** and **Count**, because the close box in this small dialogue is near the buttons.

**Comment by niksedk** [comment]
cool, some minor stuff was missing which should be fixed now - also the UpdateLanguageFiles should be fixed now, thx :)

**Pull request merged**